Makefile

```diff
@@ -6,206 +6,211 @@
+	@cat docker-compose-nvidia.yml | yq '.services.ray_server.deploy.resources.reservations.devices[0].device_ids |= (strenv(NVIDIA_VISIBLE_DEVICES) | split(",")) | ..style="double"' | \
+		EDITION=$${EDITION:=local-ce} DEFAULT_USER_UID=$$(cat ${SYSTEM_CONFIG_PATH}/user_uid) docker compose ${COMPOSE_FILES} -f - up -d --quiet-pull
+else
+	@EDITION=$${EDITION:=local-ce} DEFAULT_USER_UID=$$(cat ${SYSTEM_CONFIG_PATH}/user_uid) docker compose ${COMPOSE_FILES} up -d --quiet-pull
+endif
+
+.PHONY: latest
+latest:			## Lunch all dependent services with their latest codebase
+	@make build-latest PROFILE=${PROFILE}
+	@if [ ! -f "$$(echo ${SYSTEM_CONFIG_PATH}/user_uid)" ]; then \
+		mkdir -p ${SYSTEM_CONFIG_PATH} && \
+		docker run --rm --name uuidgen ${INSTILL_CORE_IMAGE_NAME}:latest uuidgen > ${SYSTEM_CONFIG_PATH}/user_uid; \
+	fi
+ifeq (${NVIDIA_GPU_AVAILABLE}, true)
+	@docker inspect --type=image instill/ray:latest-${RAY_PLATFORM} >/dev/null 2>&1 || printf "\033[1;33mINFO:\033[0m This may take a while due to the enormous size of the Ray server image, but the image pulling process should be just a one-time effort.\n" && sleep 5
+	@cat docker-compose-nvidia.yml | yq '.services.ray_server.deploy.resources.reservations.devices[0].device_ids |= (strenv(NVIDIA_VISIBLE_DEVICES) | split(",")) | ..style="double"' | \
+		COMPOSE_PROFILES=${PROFILE} EDITION=$${EDITION:=local-ce:latest} DEFAULT_USER_UID=$$(cat ${SYSTEM_CONFIG_PATH}/user_uid) docker compose ${COMPOSE_FILES} -f docker-compose-latest.yml -f - up -d --quiet-pull
+else
+	@COMPOSE_PROFILES=${PROFILE} EDITION=$${EDITION:=local-ce:latest} DEFAULT_USER_UID=$$(cat ${SYSTEM_CONFIG_PATH}/user_uid) docker compose ${COMPOSE_FILES} -f docker-compose-latest.yml up -d --quiet-pull
+endif
+
+.PHONY: build-latest
+build-latest:				## Build latest images for all VDP components
+	@if [ ! "$$(docker image inspect ${INSTILL_CORE_IMAGE_NAME}:latest --format='yes' 2> /dev/null)" = "yes" ]; then \
+	docker build --progress plain \
+		--build-arg ALPINE_VERSION=${ALPINE_VERSION} \
+		--build-arg GOLANG_VERSION=${GOLANG_VERSION} \
+		--build-arg K6_VERSION=${K6_VERSION} \
+		--build-arg CACHE_DATE="$(shell date)" \
+		--target latest \
+		-t ${INSTILL_CORE_IMAGE_NAME}:latest .; \
+	fi
+	@docker run --rm \
+		-v /var/run/docker.sock:/var/run/docker.sock \
+		-v ./.env:/instill-ai/core/.env \
+		-v ./docker-compose-build.yml:/instill-ai/core/docker-compose-build.yml \
+		--name ${CONTAINER_BUILD_NAME}-latest \
+		${INSTILL_CORE_IMAGE_NAME}:latest /bin/sh -c " \
+			API_GATEWAY_VERSION=latest \
+			MGMT_BACKEND_VERSION=latest \
+			PIPELINE_BACKEND_VERSION=latest \
+			CONSOLE_VERSION=latest \
+			COMPOSE_PROFILES=${PROFILE} docker compose -f docker-compose-build.yml build --progress plain \
```

Comment:
```suggestion
		${INSTILL_CORE_IMAGE_NAME}:latest /bin/sh -c " \
			API_GATEWAY_VERSION=latest \
			MGMT_BACKEND_VERSION=latest \
			PIPELINE_BACKEND_VERSION=latest \
			MODEL_BACKEND_VERSION=latest \
			CONSOLE_VERSION=latest \
			COMPOSE_PROFILES=${PROFILE} docker compose -f docker-compose-build.yml build --progress plain \
```

Reply:
we are missing `CONTROLLER_MODEL_VERSION=latest` too